tests/cupy_tests/random_tests/test_generator.py

```diff
@@ -1188,9 +1188,21 @@ def test_randint_1(self):
     def test_randint_2(self):
         self.generate(3, 4, size=(3, 2))
 
-    def test_randint_3(self):
+    def test_randint_empty1(self):
         self.generate(3, 10, size=0)
 
+    def test_randint_empty2(self):
+        self.generate(3, size=(4, 0, 5))
+
+    def test_randint_overflow(self):
+        self.generate(numpy.int8(-100), numpy.int8(100))
+
+    def test_randint_float1(self):
+        self.generate(-1.2, 3.4, 5)
+
+    def test_randint_float2(self):
+        self.generate(6.7, size=(2, 3))
```

Comment:
How about checking the value range? (not only this test, but also other `randint` tests)

Reply:
Currently such tests are only in `test_sample`.  It's similar to that `tests/cupyx_tests/test_scatter.py` has few tests of `cupyx.scatter_add` because `tests/cupy_tests/core_tests/test_ndarray_scatter.py` has many tests of `ndarray.scatter_add`.  BTW, I'm aware of #2251 but left it in the PR.

Reply:
I see, but this PR lacks output range checks for floating-point boundaries.
We need it anyway.
Maybe implement such checks in `test_sample.py`?

Reply:
I see.
I overlooked that `test_bound_float1` in `test_sample.py` is testing floating-point ranges.